enterprise/migrations/0229_enterprisecustomeruser_user_fk_and_more.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.17 on 2025-02-25 22:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
This migration already exists in the `master` branch now, right? Does this branch need a rebase?